projects/examples/src/styles.scss

```
@@ -1,5 +1,7 @@
 /* You can add global styles to this file, and also import other style files */
-@use 'tailwindcss';
+@import 'tailwindcss' source(none);
+@source './app';
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)